gui/default/syncthing/core/syncthingController.js

```diff
@@ -1091,6 +1091,19 @@ angular.module('syncthing.core')
         };
 
         $scope.editSettings = function () {
+            // Get logging debug options
+            $http.get(urlbase + '/system/debug').success(function (data) {
+                var debug = {};
+                data.enabled = data.enabled || [];
+                $.each(data.facilities, function(key, value) {
```

Comment:
Is this the same as `data.facilities.forEach`? Just asking because `$.each` isn't used in syncthingController.js anywhere else.

Reply:
Same thing. forEach I think is less cross browsery.